buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -1,7 +1,7 @@
 object Versions {
-    const val kotlin = "1.3.61"
+    const val kotlin = "1.3.71"
```

Comment:
will we bump this to 1.3.72?

Reply:
That is the plan yes. I haven't taken the time to bump it and see if nothing broken.

From what I understand, `.71` and `.72` should be 100% compatible so no worries here.